gwpy/time/tests/test_time.py

```diff
@@ -86,46 +72,108 @@ def _test_with_errors(func, in_, out):
     (Time(57754.0001, format='mjd'), LIGOTimeGPS(1167264026, 640000000)),
     (Quantity(1167264018, 's'), 1167264018),
     (Decimal('1126259462.391000000'), GW150914),
-    pytest.param(GlueGPS(GW150914.gpsSeconds, GW150914.gpsNanoSeconds),
-                 GW150914, marks=pytest.mark.requires("glue")),
+    pytest.param(
+        GlueGPS(GW150914.gpsSeconds, GW150914.gpsNanoSeconds),
+        GW150914,
+        marks=pytest.mark.requires("glue"),
+    ),
     (numpy.int32(NOW), NOW),  # fails with lal-6.18.0
     ('now', NOW),
     ('today', TODAY),
     ('tomorrow', TOMORROW),
     ('yesterday', YESTERDAY),
+])
+def test_to_gps(in_, out, use_maya):
+    """Test that :func:`to_gps` works with and without maya.
+    """
+    if use_maya:  # do nothing
+        ctx = nullcontext()
+    else:  # force 'install maya' to error to use dateutil
+        ctx = mock.patch.dict("sys.modules", {"maya": None})
+    with ctx:
+        assert time.to_gps(in_) == out
+
+
+@pytest.mark.requires("maya")
+@pytest.mark.parametrize(("in_", "out"), [
+    ('Oct 30 2016 12:34 CST', 1161887657),
+])
+def test_to_gps_maya(in_, out):
```

Comment:
What's diff between 

Test that :func:`to_gps` works with and without maya

and

Test that :func:`gwpy.time.to_gps` works with maya.